Add "How to capture client-side errors on a page" (#24)
DevExpress/testcafe-examples: examples/wait-for-file-download/README.md

**Test Code**: [test.js](test.js)

> This example shows how to specify that a test should wait for the file to download.

Reviewer: In the future, don't combine the different changes for two examples into one PR.

Author: ok

> This example shows how to specify that a test should wait for the file to download.

Reviewer 2: ```suggestion
This example shows how to program a test to wait for a file to download.
```

Before the test begins, it obtains the `downloadedFilePath`. If the file already exists, the test rewrites it without warnings or errors.

> This example runs in Google Chrome. The asynchronous `waitForDownload` function runs a `for` loop which checks if the file exists in the specified download path. The body of the loop runs multiple times with a timeout between attempts until the download finishes. If the function cannot locate the file, it returns `false`.

Reviewer 2: ```suggestion
This example runs in Google Chrome. The asynchronous `waitForDownload` function runs a `for` loop, which checks if the file exists in the specified download path. The body of the loop runs multiple times with a timeout between attempts until the download is completed. If the function cannot locate the file, it returns `false`.
```
